pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotLeadControllerRestletResource.java

```diff
@@ -211,18 +212,22 @@ public List<String> getTableNames() {
 
   @JsonIgnoreProperties(ignoreUnknown = true)
   public static class LeadControllerResponse {
-    private boolean _isLeadControllerResourceEnabled;
+    @JsonProperty("leadControllerResourceEnabled")
+    private boolean _leadControllerResourceEnabled;
+
+    @JsonProperty("leadControllerEntryMap")
     private Map<String, LeadControllerEntry> _leadControllerEntryMap;
 
     @JsonCreator
-    public LeadControllerResponse(boolean isLeadControllerResourceEnabled,
-        Map<String, LeadControllerEntry> leadControllerEntryMap) {
-      _isLeadControllerResourceEnabled = isLeadControllerResourceEnabled;
+    public LeadControllerResponse(
+        @JsonProperty("leadControllerResourceEnabled") boolean leadControllerResourceEnabled,
+        @JsonProperty("leadControllerEntryMap") Map<String, LeadControllerEntry> leadControllerEntryMap) {
+      _leadControllerResourceEnabled = leadControllerResourceEnabled;
       _leadControllerEntryMap = leadControllerEntryMap;
     }
 
-    public boolean isLeadControllerResourceEnabled() {
-      return _isLeadControllerResourceEnabled;
+    public boolean getLeadControllerResourceEnabled() {
```

Comment:
For `boolean` response type, we usually put `is` as the prefix of the method
```suggestion
    public boolean isLeadControllerResourceEnabled() {
```